.circleci/config.yml

```diff
@@ -3,39 +3,28 @@ version: 2.1
 jobs:
     "server-test":
         docker:
-            - image: circleci/python:3.7.9-node-browsers
+            - image: circleci/python:3.9.2-buster-node-browsers
```

Comment:
Updating to use py3.9 as per https://github.com/plotly/dash-core/issues/203

Reply:
For now, can't use the new CircleCI `cimg` because `CHROME_BIN` is missing and expected by Karma